src/refactor_nrepl/find/find_macros.clj

```diff
@@ -61,14 +66,14 @@
               (recur macros (reader/read opts rdr)))))))))
 
 (defn- get-cached-macro-definitions [f]
-  (when-let [[ts v] (get @cache (.getAbsolutePath f))]
+  (when-let [[ts v] (get @macro-defs-cache (.getAbsolutePath f))]
     (when (= ts (.lastModified f))
       v)))
 
 (defn- put-cached-macro-definitions [f]
   (let [defs (find-macro-definitions-in-file f)
         ts (.lastModified f)]
-    (swap! cache assoc-in [(.getAbsolutePath f)] [ts defs])
+    (swap! macro-defs-cache assoc-in [(.getAbsolutePath f)] [ts defs])
```

Comment:
This too is broken now.

Guess I didn't add any tests for the caching mechanism heh